test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -1221,6 +1222,57 @@ UCS_TEST_P(test_ucp_sockaddr_cm_private_data,
 UCP_INSTANTIATE_TEST_CASE_TLS(test_ucp_sockaddr_cm_private_data, all, "all")
 
 
+class test_ucp_sockaddr_check_lanes : public test_ucp_sockaddr {
+    unsigned rndv_lanes_num(ucp_ep_h ep)
+    {
+        unsigned num               = 0;
+        ucp_ep_config_t *ep_config = ucp_ep_config(ep);
+
+        for (ucp_lane_index_t lane_idx = 0; lane_idx < UCP_MAX_LANES;
+             ++lane_idx) {
+            if (ep_config->key.rma_bw_lanes[lane_idx] != UCP_NULL_LANE) {
+                num++;
+            }
+        }
+
+        return num;
+    }
+
+protected:
+    void check_rndv_lanes(ucp_ep_h ep)
+    {
+        for (ucp_lane_index_t lane_idx = 0;
+             lane_idx < ucp_ep_num_lanes(ep); ++lane_idx) {
+            if ((lane_idx != ucp_ep_get_cm_lane(ep)) &&
+                (ucp_ep_get_iface_attr(ep, lane_idx)->cap.flags &
+                         (UCT_IFACE_FLAG_GET_ZCOPY |
+                          UCT_IFACE_FLAG_PUT_ZCOPY)) &&
+                /* RNDV lanes are not selected for transports which don't
+                 * support UCT_MD_FLAG_INVALIDATE capability on MD.
+                 * TODO: fix by allowing CONNECT_TO_EP transports being selected
+                 *       without UCT_MD_FLAG_INVALIDATE capability */
```

Comment:
/* RNDV lanes should be selected if transport supports get/put_zcopy and invalidate */